designs/ursula/src/elastic.mjs

```diff
@@ -32,14 +34,13 @@ function ursulaElastic({
     .attr('data-text', ':')
     .attr('data-text', units(waistBandLength * store.get('elasticScale') + 2 * sa))
 
-  // Paperless?
-  if (paperless) {
-    macro('hd', {
-      from: points.elasticInfo,
-      to: points.elasticInfo,
-      y: points.elasticInfo.y + sa + 15,
-    })
-  }
+  // Draw a box around the text, so the part shows up correctly.
+  paths.box = new Path()
+    .move(new Point(-10,-10))
+    .line(new Point(-10,15))
+    .line(new Point(200,15))
+    .line(new Point(200,-10))
+    .close()
```

Comment:
Would it make sense to add a `.setClass('hidden')` here to avoid confusing the user who might assume these lines are meaningful?

It will have the same effect (taken on board for calculating the bounding box of the part) but will be hidden from the output.

Reply:
I added the `.setClass('hidden')` as suggested.